unity-client/Assets/Scripts/MainScripts/DCL/Controllers/InputController/InputController.cs

```diff
@@ -74,6 +74,10 @@ public void Update_Trigger()
         for (var i = 0; i < triggerTimeActions.Length; i++)
         {
             var action = triggerTimeActions[i];
+
+            if (action.blockTrigger != null && action.blockTrigger)
```

Comment:
Just in case I would suggest we always use the `Get()` method of our scriptable object variables to avoid confusion on Unity's "not null" check, I'd change that line into:
`if (action.blockTrigger != null && action.blockTrigger.Get())`

Reply:
Thanks for the suggestion, you're right! I have done the change in the last commit.